### PR TITLE
Add missing parameters

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/11197

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 1]
```diff
@@ -232,6 +235,90 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowedQuestionTypesInRegistrationForm
+This setting governs which users in a tenant can add which registration form questions to event registration page for attendees to answer when registering for the event.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:239]:
```suggestion
This setting governs which users in a tenant can add which registration form questions to an event registration page for attendees to answer when registering for the event.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 2]
```diff
@@ -232,6 +235,90 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowedTownhallTypesForRecordingPublish
+This setting describes how IT Admins can control which types of townhall can have their recordings published.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:256]:
```suggestion
This setting describes how IT admins can control which types of Town Hall attendees can have their recordings published.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 3]
```diff
@@ -232,6 +235,90 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowedWebinarTypesForRecordingPublish
+This setting describes how IT Admins can control which types of webinars can have their recordings published.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:273]:
```suggestion
This setting describes how IT admins can control which types of webinar attendees can have their recordings published.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 4]
```diff
@@ -232,6 +235,90 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowEventIntegrations
+This setting governs the access to the integrations tab in event creation workflow.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:290]:
```suggestion
This setting governs access to the integrations tab in the event creation workflow.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 5]
```diff
@@ -232,6 +235,90 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -TownhallChatExperience
+This setting governs if the user can enable Comment Stream chat experience for Townhalls.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:307]:
```suggestion
This setting governs whether the user can enable the Comment Stream chat experience for Town Halls.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 6]
```diff
@@ -232,6 +235,95 @@ Accept pipeline input: False
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowedWebinarTypesForRecordingPublish
+This setting describes how IT admins can control which types of webinar attendees can have their recordings published.
+
+Possible values are:
+None,InviteOnly,EveryoneInCompanyIncludingGuests,Everyone.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:278]:
```suggestion
None, InviteOnly, EveryoneInCompanyIncludingGuests, Everyone.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @get-itips, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**